### PR TITLE
new docs: two more commands

### PR DESCRIPTION
**Pull request by hengm3467**
Docs for create index and create mv

### CODE REVIEW

File: docs/sql/commands/sql-select.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ Use the `SELECT` command to retrieve rows from a table or materialized view. It
 SELECT [ ALL | DISTINCT] [ * | <expression> [ AS <output_name> ] [, <expression> [ AS <output_name> ]...] ]
     [ FROM <from_item> [, <from_item> ...] ]
     [ WHERE <condition> ]
-    [ GROUP BY <grouping_expression> [, <grouping_expression>... ]
+    [ GROUP BY <grouping_expression> [, <grouping_expression>... ]]
     [ HAVING <condition> ]
     [ ORDER BY <sort_expression> [ ASC | DESC ] [, ...] ]
-    [ LIMIT { count | ALL } ]
+    [ LIMIT <count> ]
     [ OFFSET start [ ROW | ROWS ] ]
```
Comment (CharlieSYH) [inline docs/sql/commands/sql-select.md]: Maybe add ; at the end of the statement.

File: docs/sql/commands/sql-create-index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+id: sql-create-index
+title: CREATE INDEX
+description: Create an index
```
Comment (CharlieSYH) [inline docs/sql/commands/sql-create-index.md]: Period at the end.

File: docs/sql/commands/sql-create-mv.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+description: Create a materialized view
+slug: /sql-create-mv
+---
+
+Use the `CREATE MATERIALIZED VIEW` command to create a materialized view. A materialized view can be created based on tables or materialized views.
+
+## Syntax
+
+```sql
+CREATE MATERIALIZED VIEW <mv_name> AS <select_query>;
```
Comment (CharlieSYH) [inline docs/sql/commands/sql-create-mv.md]: I thought we agreed to drop "_name" if the parameter is a name.

**Review by CharlieSYH — APPROVED** [review]

File: docs/sql/commands/sql-create-table.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 id: sql-create-table
 title: CREATE TABLE
-description: Create a table
+description: Create a table.
 slug: /sql-create-table
 ---
```
Comment (CharlieSYH) [inline docs/sql/commands/sql-create-table.md:7]: e2e:
Create a table under the schema.
statement ok
create table ddl_schema.ddl_table;

Users can create a table under a specific schema. Maybe add that to the syntax?

**Review by CharlieSYH — APPROVED** [review]

File: docs/sql/commands/sql-create-table.md (hunk 4 above)
Comment (hengm3467) [reply docs/sql/commands/sql-create-table.md:7]: I'll add a note about the schema.

**Pull request merged**